src/sentry/static/sentry/app/views/releases/detail/organization/releaseAllEvents.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {Link} from 'react-router';
+
+import SentryTypes from 'app/sentryTypes';
+import GroupList from 'app/components/groupList';
+import {t} from 'app/locale';
+
+export default class ReleaseAllEvents extends React.Component {
+  static contextTypes = {
+    release: SentryTypes.Release,
+  };
+
+  render() {
+    let {orgId} = this.props.params;
+    return (
+      <div>
+        <div className="alert alert-block">
```

Comment:
Can we clean this up to use `<Alert>`?

Reply:
It'll look a little different but we can update it